examples/v2/chat2.nim

```diff
@@ -171,9 +181,10 @@ proc processInput(rfd: AsyncFD, rng: ref BrHmacDrbgContext) {.async.} =
     let payload = cast[string](message.payload)
     echo &"{payload}"
     info "Hit subscribe handler", topic=topic, payload=payload, contentTopic=message.contentTopic
-  await node.subscribe(topic, handler)
 
-  var chat = Chat(node: node, transp: transp, subscribed: true, connected: false, started: true)
+  # XXX Timing issue with subscribe, need to wait a bit to ensure GRAFT message is sent
+  await sleepAsync(5.seconds)
```

Comment:
@dryajov is there no more efficient way to get info from libp2p when everything was successful?

Reply:
This is out of scope of this PR. This fixes a blocking issue. See https://github.com/status-im/nim-waku/issues/227

Reply:
This is also application code (example chat), not library code. The goal is to support the critical path of Dingpu